.gitmodules

```diff
@@ -25,7 +25,16 @@
 	path = FreeRTOS/backoffAlgorithm
 	branch = main
 	url = https://github.com/FreeRTOS/backoffAlgorithm.git
-
 [submodule "FreeRTOS/FreeRTOS-Kernel"]
 	path = FreeRTOS/FreeRTOS-Kernel
 	url = https://github.com/FreeRTOS/FreeRTOS-Kernel.git
+[submodule "aws/jobs-for-aws-iot-embedded-sdk"]
+	path = aws/jobs-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/Jobs-for-AWS-IoT-embedded-sdk.git
+	branch = main
```

Comment:
is branch required?

Reply:
Other library submodules have the branch name mentioned, so this is just being consistent with them.
In the future, if we add automation functionality of auto-updating the submodules, the tracking branch will be useful